### DISCUSSION TITLE
Limit the data you show on the open stats (such as Google keywords)

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#221] [Anything Else]
"Love the open traffic chart. However, would there be a way to limit how much data the public chart sees, versus what I do? E.g. I'd love to publish the stats, but not necessarily my Google keyword stats"

feedback via email

**Comment by metmarkosaric** [comment]
"Feature request: for users who make their dashboards public it would be cool if we could have an embed for *just* the Top Pages part."

(via [twitter](https://twitter.com/Chronotope/status/1514664471419101187))

**Reply by corneliusroemer** [reply]
Related: #2198, though that request is for such a limit to apply more broadly, not just to open stats.
